<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleNetUI</class>
 <widget class="QMainWindow" name="SimpleNetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleNetUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="piclabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="labelLineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="predictionLineEdit"/>
     </item>
     <item>
      <widget class="QSpinBox" name="labelSpinBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Learning rate 1e-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Regularization</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="regBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Batch size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="bsBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Epochs</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="iterBox"/>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="learningRateBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>261</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_13">
     <item>
      <widget class="QLabel" name="lossLabel">
       <property name="text">
        <string>Loss:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="accLabel">
       <property name="text">
        <string>Accuracy:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>91</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_16">
     <item>
      <widget class="QPushButton" name="buttonNormalizationReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonMeanImage">
       <property name="text">
        <string>MeanSubtr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNormalize">
       <property name="text">
        <string>Normalize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonStandardize">
       <property name="text">
        <string>Standardize</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_17">
     <item>
      <widget class="QLabel" name="dataMax">
       <property name="text">
        <string>Max:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dataMin">
       <property name="text">
        <string>Min:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>101</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_18">
     <item>
      <widget class="QLabel" name="labelMeanImage">
       <property name="text">
        <string>MeanImage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stdImage">
       <property name="text">
        <string>stdImage</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>240</y>
      <width>801</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Activations</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="pic_hidden">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>781</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="pic_softmax">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="lossplot" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>401</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="accplot" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>360</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>254</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Train</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>171</width>
      <height>205</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Weight analysis</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QLabel" name="labelWMax">
            <property name="text">
             <string>WMax: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelWMin">
            <property name="text">
             <string>WMin: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labeldWMax">
            <property name="text">
             <string>dWMax:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labeldWMin">
            <property name="text">
             <string>dWMin:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUpdMax">
            <property name="text">
             <string>UpdMax:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUpdMin">
            <property name="text">
             <string>UpdMin:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRatio">
            <property name="text">
             <string>Ratio:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="updWeightViz">
       <property name="text">
        <string>Update weight viz</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pushButton</zorder>
   <zorder>stopButton</zorder>
   <zorder>resetButton</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>updWeightViz</zorder>
   <zorder>lossplot</zorder>
   <zorder>accplot</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1224</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_dataset"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_dataset">
   <property name="text">
    <string>Open dataset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
